openfe_gromacs/tests/conftest.py

```diff
@@ -142,6 +143,15 @@ def T4_protein_component():
     return comp
 
 
+@pytest.fixture(scope="session")
+def alanine_dipeptide_component():
+    with resources.files("openfe_gromacs.tests.data") as d:
```

Comment:
This part doesn't work yet, that's why for now I've been using the test data stored in the openfe repo (e.g. `T4_protein_component` in `resources.files("openfe.tests.data")`).

Reply:
I've pushed a commit to fix this, you were missing an `__init__.py` file in the `tests/data` directory.